src/__experimental__/components/label/label.component.js

```diff
@@ -10,14 +12,27 @@ const Label = ({
   helpId,
   helpTag,
   helpTabIndex,
+  hasError,
+  hasWarning,
+  hasInfo,
+  tooltipMessage,
   ...props
 }) => (
   <LabelStyle
     data-element='label'
+    hasError={ hasError }
     { ...props }
   >
     {children}
-    {help && (
+
+    {(hasError || hasWarning || hasInfo) && tooltipMessage && (
+      <ValidationIcon
+        type={ getValidationType({ hasError, hasWarning, hasInfo }) }
+        tooltipMessage={ tooltipMessage }
+      />
+    )}
+
+    {!hasError && !hasWarning && !hasInfo && help && (
```

Comment:
If you do the above, cuts down on some repetition as you can do `!validationsPresent(props) && …` here